test.sql

```diff
@@ -0,0 +1,229 @@
+    WITH EventData AS (
```

Comment:
oh I think this and the other files were mistakenly checked in?

Reply:
Fixed, sorry about that